File: Iniciante/1179.py
pares = []
impares = []

for i in range(15):
  valor = int(input())
  
  if (valor % 2 == 0):
    pares.append(valor)
  else:
    impares.append(valor)

  if (len(pares) == 5):
    for j in range(len(pares)):
      print("par[{}] = {}".format(j, pares[j]))

    pares = []
  elif (len(impares) == 5):
    for k in range(len(impares)):
      print("impar[{}] = {}".format(k, impares[k]))

    impares = []

for i in range(len(impares)):
  print("impar[{}] = {}".format(i, impares[i]))

for i in range(len(pares)):
  print("par[{}] = {}".format(i, pares[i]))

File: Iniciante/1179.js
let pares = [], impares = [];

for (let i = 0; i < 15; i++) {
  let valor = Number(lines.shift());
  
  if (valor % 2 === 0) {
    pares.push(valor);
  } else {
    impares.push(valor);
  }

  if (pares.length === 5) {
    for (let j = 0; j < pares.length; j++) {
      console.log(`par[${j}] = ${pares[j]}`);
    }
    pares = [];
  } else if (impares.length === 5) {
    for (let k = 0; k < impares.length; k++) {
      console.log(`impar[${k}] = ${impares[k]}`);
    }
    impares = [];
  }
}

for (let i = 0; i < impares.length; i++) {
  console.log(`impar[${i}] = ${impares[i]}`);
}

for (let i = 0; i < pares.length; i++) {
  console.log(`par[${i}] = ${pares[i]}`);
}
